apis/vshn/v1/dbaas_vshn_postgresql.go

```diff
@@ -147,6 +155,16 @@ type VSHNPostgreSQLBackup struct {
 	// +kubebuilder:default=6
 	// +kubebuilder:validation:XIntOrString
 	Retention int `json:"retention,omitempty"`
+
+	// DeletionProtection will protect the instance from being deleted for the given retention time.
+	// This is enabled by default.
+	// +kubebuilder:default=true
```

Comment:
Should we really default to `true`?

Honest question I don't know. I see that we want sane default settings for prod, but I think it's a bit unexpected to have this by default.

Reply:
I believe this should be set by default :)